### ISSUE TITLE
去掉sentence loss，请问是否尝试SpanBert中 Span Boundary Objective目标loss，或者DOC-SENTENCES？以及采用 bytes-level 的 BPE？

### ISSUE THREAD

**Issue by sliderSun** [brightmart/roberta_zh#40]
total_loss = masked_lm_loss # TODO remove next sentence loss 2019-08-08, + next_sentence_loss

**Comment by brightmart** [comment]
这个代码刚刚更新了，使用了albert中的文本段连续性的损失了。

**Comment by brightmart** [comment]
去掉nsp并使用doc-sentences的方式，在roberta_zh项目有尝试，效果还可以。

这次先会看看albert的文本段连续性任务的效果
